### ISSUE TITLE
jsgeoda: integrate jsgeoda test

### ISSUE THREAD

**Issue by lixun910** [GeoDaCenter/reactgeoda#19]
some tasks:
- try to convert GeoJSON object (loaded by loaders.gl) into ArrayBuffer object for jsgeoda
- try to create a Local Moran map using e.g. guerry dataset https://geodacenter.github.io/data-and-lab/Guerry/
- try to create a Moran Scatter plot

**Comment by lixun910** [comment]
update from @justin-kleid:

I merged my choropleth branch into the main branch. For the choropleth map, I am tracking two data states; one for the Kepler processed data, and one for raw data for jsgeoda (ideally this will be consolidated to just tracking the processed data once we update jsgeoda). The choropleth map works for uploaded geojson's where the user can select variables and settings; I have tested it out on a downloaded version of the nat regimes dataset (https://webgeoda.github.io/data/natregimes.geojson). 

I also added the jenksCategory column to the Json data object before its processed by kepler, and it seems to work since its field is given the valueAccessor attribute. However, if I remove the custom valueAccessor function that I put with the layerVisualChannelConfigChange action, the code still breaks with the "func is not a function" error. Not super sure what the bug is, but I would appreciate any advice for the code.
